### DISCUSSION TITLE
Demo Sites | 示例站点

### DISCUSSION THREAD

**Discussion by MaLuns** [MaLuns/hexo-theme-async#7] [Show 分享展示]
**不在手动增加 Demo Sites，请直接修改 [sites.json](https://github.com/MaLuns/hexo-theme-async/blob/dev/docs/.vitepress/assets/sites.json) 创建 PR**

~~欢迎在此处分享展示您使用本主题搭建的最终站点！您的站点将可能被选为示例站点，并展示于主题文档中。~~

~~您可以选择 `Show and tell` 这一分类并新建一个 `discussion`，而不必在此帖下回复。~~

### 提交 PR
可以直接修改 [sites.json](https://github.com/MaLuns/hexo-theme-async/blob/dev/docs/.vitepress/assets/sites.json) 创建 PR。

其他你想要说的话（畅所欲言～）
---
那么向大家展示你的站点吧～

**Comment by littlefoodiedy** [comment]
https://littlefoodiedy.github.io/

**Comment by wenzhuo4657** [comment]
wenzhuo4657.org

**Comment by XINBOY722** [comment]
https://xinboy722.github.io/
感谢大大！让我完成了第一个自己的博客！
![微信截图_20250331151841](https://github.com/user-attachments/assets/07082bff-b3f4-48d6-bcdb-e5c4a378ccf9)

**Reply by MaLuns** [reply]
啊哈，墨迹天气的这视频 CDN 又好了啊 🤔

**Reply by XINBOY722** [reply]
是哒——

**Comment by caijinhong1999** [comment]
https://www.jinhongcai.work  欢迎来踩~
